factory/crypto/cryptoComponents.go

```diff
@@ -294,10 +294,10 @@ func (ccf *cryptoComponentsFactory) createCryptoParams(
 			return nil, fmt.Errorf("invalid node configuration: import-db mode and allValidatorsKeys.pem file provided")
 		}
 
-		return ccf.generateCryptoParams(keygen, "in import mode", handledPrivateKeys)
+		return ccf.generateCryptoParams(keygen, "in import mode", make([][]byte, 0))
```

Comment:
because of L293 we are sure this line will be called with an empty slice. Can be either reverted or left as it is.